python36/Dockerfile

```diff
@@ -9,4 +9,5 @@ RUN yum -y install python36 python36-devel python36-virtualenv \
     git docker vim jq \
     openldap-devel gcc-c++ \
     && yum clean all \
-    && pip-3.6 install --no-cache-dir awscli aws-sam-cli pipenv pipfile-requirements
+    && pip-3.6 install --no-cache-dir awscli aws-sam-cli pipenv pipfile-requirements \
+    && pip install --upgrade pip
```

Comment:
if we use pip here, I think it will use the Python 2.7's pip. Can try pip -V to check.
Should be pip-3.6 install -upgrade pip. After the upgrade, please try pip -V (instead of pip-3.6 -V) to check the difference.

Reply:
pipenv requires a newer version of pip, pip will be upgraded automatically without add a line